restapi/src/main/java/io/stargate/web/resources/v2/schemas/TablesResource.java

```diff
@@ -292,26 +290,32 @@ public Response updateTable(
         () -> {
           DataStore localDB = db.getDataStoreForToken(token);
 
-          String tableOptions;
-          try {
-            tableOptions = Converters.getTableOptions(tableUpdate);
-          } catch (Exception e) {
+          TableOptions options = tableUpdate.getTableOptions();
+          List<ClusteringExpression> clusteringExpressions = options.getClusteringExpression();
+          if (clusteringExpressions != null && !clusteringExpressions.isEmpty()) {
             return Response.status(Response.Status.BAD_REQUEST)
                 .entity(
                     new Error(
-                        "Unable to create table options " + e.getMessage(),
+                        "Cannot update the clustering order of a table",
                         Response.Status.BAD_REQUEST.getStatusCode()))
                 .build();
           }
 
+          Integer defaultTTL = options.getDefaultTimeToLive();
+          if (defaultTTL == null) {
+            return Response.status(Response.Status.BAD_REQUEST)
+                .entity(
+                    new Error("No update provided", Response.Status.BAD_REQUEST.getStatusCode()))
+                .build();
+          }
+
           localDB
-              .query(
-                  String.format(
-                      "ALTER TABLE %s.%s %s",
-                      Converters.maybeQuote(keyspaceName),
-                      Converters.maybeQuote(tableName),
-                      tableOptions),
-                  ConsistencyLevel.LOCAL_QUORUM)
+              .queryBuilder()
```

Comment:
What about `clusteringExpressions`?

Reply:
You can't have any. You cannot change the clustering order of a table, it's not valid CQL syntax at all. See above, the patch adds code to reject a query that sets `clusteringExpressions`.